examples/nixnet/flexray-signal-single-point-input.py

```diff
@@ -100,8 +100,9 @@ def check_for_error(status):
     )
     check_for_error(set_property_response.status)
 
+    # To get more meaningful data use this with output example
     while i < 10:
-        # Update the signal data
+        # Update the signal data, the first read normally return the default payload values.
```

Comment:
Revert this. I think the comment on 103 (rephrased statement) would convey this too.

Reply:
Done